receiver/opencensus/ocmetrics/opencensus.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocmetrics
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"google.golang.org/api/support/bundler"
+
+	"go.opencensus.io/trace"
+
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	agentmetricspb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/metrics/v1"
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	"github.com/census-instrumentation/opencensus-service/internal"
+	"github.com/census-instrumentation/opencensus-service/metricsink"
+)
+
+// Receiver is the type used to handle metrics from OpenCensus exporters.
+type Receiver struct {
+	metricSink         metricsink.Sink
+	metricBufferPeriod time.Duration
+	metricBufferCount  int
+}
+
+// New creates a new opencensus.Receiver reference.
```

Comment:
`// New creates a new opencensus.Receiver`
reference is not too correct :)

Reply:
Good catch, done.